### DISCUSSION TITLE
Does PasswordPusher allow logins through e.g. SAML federation?

### DISCUSSION THREAD

**Discussion by jbtwelve** [pglombardo/PasswordPusher#888] [Ideas]
As I understand it, logins are only local today . Is there a way to use a 3rd party IdP through e.g. federation?

**Comment by pglombardo** [comment]
Hi @jbtwelve - not yet.

I've begun research in this area and have had multiple requests for this (and SSO) but haven't gotten to adding support yet.  I'll leave this topic open and update once I make progress on this front.
